src/obr/core/core.py

```diff
@@ -226,31 +226,31 @@ def get_timestamp_from_log(log) -> str:
     return log_name.replace(before + "_", "")
 
 
+def find_tags(path: Path, tags: list, tag_mapping):
+    """Recurses into subfolders of path until a system folder is found
+
+    Returns:
+      Dictionary mapping paths to tags -> tag
+    """
+    _, folder, _ = next(os.walk(path))
+    is_case = len(folder) == 0
```

Comment:
Very nice, this will solve #176

Reply:
This is currently the "old" approach, which is also brittle, because it assumes that the archived folder does not contain any subfolders. However, this assumptions does not necessarily hold true.